src/Badge/Service/ClientStrategy.php

```diff
@@ -28,72 +26,65 @@ public function __construct(GithubClient $githubClient, BitbucketClient $bitbuck
         $this->bitbucketClient = $bitbucketClient;
     }
 
+    /**
+     * @throws SourceClientNotFound
+     * @throws \Http\Client\Exception
```

Comment:
please use short annotation ad add import for the class. Thanks!

Reply:
Done..